assets/src/dashboard/app/views/editorSettings/googleAnalytics/index.js

```diff
@@ -40,20 +47,48 @@ const TEXT = {
 };
 // todo add link
```

Comment:
What is this comment for?

Reply:
the text to display here has a link in it to a resource to help set up GA, I didn't yet tackle how to do the translation bit WITH the link in here so i left myself a comment since i am trying to keep these settings PRs bite size. :) I can expand on it though. It won't be there very much longer.